UnitTests/UnitTests.csproj

```diff
@@ -1,9 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
```

Comment:
Please set this back to net461;netcoreapp2.0 as we need to explicitly test multiple framework versions to ensure full test coverage

Reply:
Ah, I got this mixed up with .NET Standard and .NET Fx support. I'll add back the net461 target framework. Thanks!!

Reply:
OK, so to clarify the situation: I forgot that unit test projects must be .NET Fx based or .NET Core based and got confused with .NET Standard vs. .NET Core. I should've known better--but hey, it was late! 😄 

I should not have removed the `net461` TFM, as you indicated. That's needed to ensure that when using the libraries with a .NET Fx 4.6.1-based project that everything works OK. So I will put that back in.

However, the presence of the `netcoreapp2.0` TFM results in a compiler warning with respect to the NUnit Test Adapter. Updating this TFM to `netcoreapp2.1` removes this warning. And according to [Microsoft](https://docs.microsoft.com/en-us/dotnet/core/versions/selection), minor version updates within a major version of .NET Core are compatible with one another. So it should not be an issue to leave this as `netcoreapp2.1` (or even `netcoreapp2.2`, but `netcoreapp2.1` was the minimum necessary to remove the warning).

Reply:
See 330b6c5 which reintroduces `net461`.

Reply:
I’m cool with this - can you update the acceptance test csproj to 2.1 then as well?

Reply:
Sure thing.